src/types/Comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
```

Comment:
It's a great start, but the task requires you to add a user and comments to each post. You might want to add a `User` type and include it in your `Post` type. Keep up the good work! 😄